packages/ai/src/ai/web/endpoints/schedules.py

```
@@ -0,0 +1,211 @@
+# SOFTWARE.
+# =============================================================================
+
+import logging
+
+from ai.web import Body, Request, Result, error, exception, response
+from ai.web.scheduler.models import ScheduleCreate, ScheduleList
+
+logger = logging.getLogger(__name__)
+
+
+def _get_scheduler(request: Request):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_get_scheduler`

(ANN202)